services/web-app/lib/github.js

```diff
@@ -490,6 +490,46 @@ export const getLibraryRelatedLibs = async (libData) => {
   return relatedLibs
 }
 
+/**
+ * Compiles an object list of all known tags
```

Comment:
Is this "tags" as in framework tags?

Reply:
hahaah copy/paste got me, I need to update this description. this function just compiles an array of other assets related to a given asset (other frameworks)

Reply:
Done!